src/NextBuilds.php

```diff
@@ -191,6 +216,20 @@ function (Event $event) {
         );
     }
 
+    /**
+     * @return void
+     */
+    protected function initializeElementStatusEvents(): void
+    {
+        Event::on(Elements::class, Elements::EVENT_BEFORE_SAVE_ELEMENT, [ElementStatusEvents::class, 'rememberPreviousStatus']);
```

Comment:
Registers the new behavior (remember previous state) to the entry and persist this info in cache